### ISSUE TITLE
Scrolls server crash on preview network

### ISSUE THREAD

**Issue by GrzegorzDrozda** [txpipe/scrolls#121]
Hi 
I have problem with scrolls (version 0.4.3) on preview network. Scrolls is running from docker image and after processing block: `1702a60228ef0fd782d8428e8fe44297960f3355eea0b1eb1ce6e5818b2ea877` it is stopping. Below logs:
```
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::transport] handshake output: Accepted(10, VersionData { network_magic: 2, initiator_and_responder_diffusion_mode: false })
[2023-01-25T13:26:15Z INFO  scrolls::sources::utils] found existing cursor in storage plugin: Specific(7995187, "1702a60228ef0fd782d8428e8fe44297960f3355eea0b1eb1ce6e5818b2ea877")
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] chain-sync intersection is (7995187, 1702a60228ef0fd782d8428e8fe44297960f3355eea0b1eb1ce6e5818b2ea877)
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] requesting next block
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] requesting next block
[2023-01-25T13:26:15Z WARN  scrolls::reducers::worker] rollback requested for (7995187, 1702a60228ef0fd782d8428e8fe44297960f3355eea0b1eb1ce6e5818b2ea877)
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] requesting next block
[2023-01-25T13:26:15Z ERROR gasket::runtime] STAGE: reducers, WORK, PANIC: cbor error: Invalid CBOR structure: unexpected type indefinite bytes at position 1831: expected bytes (definite length)
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] requesting next block
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] requesting next block
...
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] chain-sync reached the tip of the chain
[2023-01-25T13:26:15Z INFO  scrolls::sources::n2n::chainsync] awaiting next block (blocking)
[2023-01-25T13:26:16Z INFO  scrolls::daemon] Scrolls is stopping...
[2023-01-25T13:26:16Z WARN  scrolls::daemon] dismissing stage: n2n with state Alive(Working)
[2023-01-25T13:26:16Z WARN  scrolls::daemon] dismissing stage: enrich-skip with state Alive(Working)
[2023-01-25T13:26:16Z WARN  scrolls::daemon] dismissing stage: reducers with state Alive(StandBy)
[2023-01-25T13:26:16Z WARN  scrolls::daemon] dismissing stage: redis with state Alive(Working)
```

**Comment by SebastienGllmt** [comment]
FYI these two are the same: https://github.com/txpipe/oura/issues/526

**Comment by jmhrpr** [comment]
closed by #122
